js/src/api/transport/ws/ws.js

```diff
@@ -194,31 +202,66 @@ export default class Ws extends JsonRpcBase {
     }, 50);
   }
 
+  _extract = (result) => {
+    // initial pubsub ACK
+    if (result.id && this._messages[result.id].subscription) {
+      this._messages[result.id].method === 'eth_subscribe'
```

Comment:
Assigning `this._messages[result.id]` to a variable would help in readability of the whole method.